code/modules/mob/language.dm

```diff
@@ -286,7 +277,6 @@
 	colour = "grey"
 	space_chance = 35
 	native = 1
-	flags = RESTRICTED
```

Comment:
Martian language's supposed to be unknown to AI. Please fix.

Reply:
Is that so?